apps/site/lib/site_web/controllers/trip_plan_controller.ex

```diff
@@ -61,7 +67,9 @@ defmodule SiteWeb.TripPlanController do
   end
 
   def to(conn, %{"address" => address}) do
-    case TripPlan.geocode(address) do
+    updated_address = Geocode.check_address(address, @options)
```

Comment:
Just noting here that we might be making extra geolocation calls here and it might have an impact on performance.

Reply:
So it looks like `check_address` will result in another `geocode` call only if this 'address' happens to actually be a parsable lat/lon string... luckily that's not a case we expect to encounter often here so I'm hopeful it will be OK performance-wise :pray: